src/Maestro/Maestro.Contracts/MergePolicyEvalutationResult.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+
+namespace Maestro.Contracts
+{
+    public class MergePolicyEvaluationResults
+    {
+        public MergePolicyEvaluationResults(IEnumerable<MergePolicyEvaluationResult> results)
+        {
+            Results = results.ToImmutableList();
+        }
+
+        public IReadOnlyList<MergePolicyEvaluationResult> Results { get; }
```

Comment:
I wonder if returning IImmutalbeList is better. It's most explicit about the contract. (Just because something is IReadOnly doesn't mean it won't change, so it's not really safe to assume it won't.  IImmutable is a much "stronger" contract that lets the caller do a lot more safely)